index.html

```diff
@@ -460,31 +460,30 @@ <h2>
         </p>
       </div>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn>within scope</dfn> of
-        <a>navigation scope</a> <var>scope</var> if the following algorithm
-        returns <code>true</code>:
+        A [=URL=] |target:URL| is said to be <dfn data-dfn-for="URL"
+        data-export="">within scope</dfn> of [=URL=] |scope:URL| if the
+        following algorithm returns `true`:
       </p>
-      <ol>
-        <li>Let <var>scopePath</var> be the [=string/concatenation=] of
-        <var>scopes</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-        U+002F (/).
+      <ol class="algorithm">
+        <li>If |target| and |scope| are not [=same origin=], return `false`.
+        </li>
+        <li>Let |scopePath:string| be the [=string/concatenation=] of
+        |scopes|'s [=URL/path=] and U+002F (/).
         </li>
-        <li>Let <var>targetPath</var> be the [=string/concatenation=] of <var>
-          target</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-          U+002F (/).
+        <li>Let |targetPath:string| be the [=string/concatenation=] of
+        |target|'s [=URL/path=] and U+002F (/).
         </li>
-        <li>If <var>target</var> is <a>same origin</a> as <var>scope</var> and
-        <var>targetPath</var> starts with <var>scopePath</var>, return
-        <code>true</code>.
+        <li>If |targetPath| starts with |scopePath:string|, return `true`.
         </li>
-        <li>Otherwise, return <code>false</code>.
+        <li>Otherwise, return `false`.
         </li>
       </ol>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn data-lt=
-        "within-scope-manifest">within scope of a manifest</dfn>
-        <var>manifest</var> if <var>target</var> is <a>within scope</a> of the
-        navigation scope of <var>manifest</var>.
+        A [=URL=] |target:URL| is said to be <dfn data-export="" data-dfn-for=
+        "manifest" data-lt="within scope">within the navigation scope of a web
+        manifest</dfn> {{WebAppManifest}} |manifest:WebAppManifest| if [=URL=]
+        |target:URL| is [=manifest/within scope=] of the |manifest|'s
+        {{WebAppManifest/scope}} member (once resolved).
```

Comment:
"once resolved" -- this is a colloquial term (URL "resolution") but I don't think it's defined anywhere.

Can you say "once [=processed=]" (linking to "processing a manifest"), because that's really what you're saying: we're talking about the scope of the manifest after processing.

Better yet, move this to the definition of "navigation scope of a manifest" above. That is:

> The navigation scope of a manifest _manifest_ is _manifest_[scope], once manifest has been processed.

or

> The navigation scope of a **processed manifest** _manifest_ is _manifest_[scope].

Then, here, you should say "of the navigation scope of _manifest_" rather than "of the _manifest_'s scope member".